src/pages/Pokemon/pokemon.css

```diff
@@ -450,6 +450,109 @@ h1 {
   width: 10%;
   color: white;
 }
+.weakness-card {
+  display: flex;
+  border-radius: 3px;
+  line-height: 18px;
+  /* max-width: 110px; */
+  margin: 0 1.5625% 0 0;
+  /* width: 38.4375%; */
+  width: 100%;
+  float: left;
+  font-size: 11px;
+  text-align: center;
+  text-transform: capitalize;
+}
+
+.grass {
+  background-color: #9bcc50;
+  color: #212121;
+}
+
+.poison {
+  background-color: #b97fc9;
+  color: white;
+}
+
+.fire {
+  background-color: #fd7d24;
+  color: white;
+}
+
+.water {
+  background-color: #4592c4;
+  color: white;
+}
+
+.flying {
+  background: linear-gradient(180deg, #3dc7ef 50%, #bdb9b8 50%);
+  color: #212121;
+}
+
+.bug {
+  background-color: #729f3f;
+  color: white;
+}
+
+.normal {
+  background-color: #a4acaf;
+  color: #212121;
+}
+
+.electric {
+  background-color: #eed535;
+  color: #212121;
+}
+
+.ground {
+  background: linear-gradient(180deg, #f7de3f 50%, #ab9842 50%);
+  color: #212121;
+}
+
+.fairy {
+  background-color: #fdb9e9;
+  color: #212121;
+}
+
+.fighting {
+  background-color: #d56723;
+  color: white;
+}
+
+.psychic {
+  background-color: #f366b9;
+  color: white;
+}
+
+.rock {
+  background-color: #a38c21;
+  color: white;
+}
+
+.steel {
+  background-color: #9eb7b8;
+  color: #212121;
+}
+
+.ice {
+  background-color: #51c4e7;
+  color: #212121;
+}
+
+.ghost {
+  background-color: #7b62a3;
+  color: white;
+}
+
+.dragon {
+  background: linear-gradient(180deg, #53a4cf 50%, #f16e57 50%);
+  color: white;
+}
+
+.dark {
+  background-color: #707070;
+  color: white;
+}
```

Comment:
mate please delete this evil, since we have these classes already defined in another css file, and these will overwrite them all and spread the corruption. We can probs use the types badge component for the weaknesses mate